src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/IdeTestAssemblyRunner.cs

```diff
@@ -34,8 +34,6 @@ protected override async Task<RunSummary> RunTestCollectionAsync(IMessageBus mes
             var completedTestCaseIds = new HashSet<string>();
             try
             {
-                ExecutionMessageSink.OnMessage(new TestAssemblyStarting(testCases, TestAssembly, DateTime.Now, GetTestFrameworkEnvironment(), GetTestFrameworkDisplayName()));
```

Comment:
📝 This is already reported by the caller. The additional message here resulted in a harness exception:
https://dev.azure.com/dnceng/public/_build/results?buildId=1256507

```xml
      <test name="Microsoft.CodeAnalysis.Testing.CreateProjectTests.CreateAnalyzerFromCSharpTemplateAsync (VS2017)" type="Microsoft.CodeAnalysis.Testing.CreateProjectTests" method="CreateAnalyzerFromCSharpTemplateAsync" time="0" result="Fail">
        <failure exception-type="System.InvalidOperationException">
          <message><![CDATA[System.InvalidOperationException : Test did not run due to a harness failure.\r\n---- System.InvalidOperationException : Duplicate attribute.]]></message>
          <stack-trace><![CDATA[
----- Inner Stack Trace -----

Server stack trace: 
   at System.Xml.Linq.XElement.AddAttributeSkipNotify(XAttribute a)
   at System.Xml.Linq.XContainer.AddContentSkipNotify(Object content)
   at MessageSinkMessageExtensions.Dispatch[TMessage](IMessageSinkMessage message, HashSet`1 messageTypes, MessageHandler`1 callback) in C:\Dev\xunit\xunit\src\xunit.runner.utility\Extensions\MessageSinkMessageExtensions.cs:line 39
   at System.Runtime.Remoting.Messaging.StackBuilderSink._PrivateProcessMessage(IntPtr md, Object[] args, Object server, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.SyncProcessMessage(IMessage msg)

Exception rethrown at [0]: 
   at System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)]]></stack-trace>
        </failure>
      </test>
```